### ISSUE TITLE
Installation instructions confusing:

### ISSUE THREAD

**Issue by jpmorris** [jalvesaq/Nvim-R#39]
When a user wants to install vim-r-plugin they google and go to:

http://www.vim.org/scripts/script.php?script_id=2628

These instructions point to the install instructions:

https://raw.githubusercontent.com/jcfaria/Vim-R-plugin/master/doc/r-plugin.txt

When I go through the install instructions it says
* Install R > 3.0.0 CHECK
* vimcom = 1.3-1

Ok I go to the link and it directs me to the release page which is a repo that is marked deprecated:
https://github.com/jalvesaq/VimCom/releases

which in turn diects me to this repo
So is installing github Nvim-R all that is needed now?
No need to install Rtools?  No need to use devtools and install vimcom (the old webpage which had these instructions has vanished).

The info on this repo says we can install Nvim-R via Vundle.  But if I do that, I take it that vim-r-plugin is not installed?  But it also says that Nvim-R is an offshoot of vim-r-plugin.  So it DOES include the vim-r-plugin?  If so if I install Nvim-R it WILL include vim-r-plugin and I WON'T have to install via downloading the vmb file from vim.org and installing with so %?  So I don't need to do anything from the R console then?

Sorry if the above is confusing, but the instructions on this repo, githubuserconent.com and vim.org are confusing so I don't know how to understand it.  

1) Perhaps the instructions can be clearified?
2) If not, can someone post the proper vim installation instructions on windows FROM SCRATCH here?

Thanks

**Comment by jalvesaq** [comment]
I will release both Nvim-R.vmb and nvimcom.zip at [Nvim-R/releases] later today (I don have time now).

Until Cream releases new GVim binary, you have two options:

   - Install Vim-R-plugin 1.2.8 and vimcom 1.2-8
   - Build Vim yourself and install Nvim-R (via Vundle).

I guess that Rtools is enough to buld Vim. I don't know for sure what I have installed in my Windows partition, but I seems that only `gcc` and `make` are required. I build Gvim with the following procedure:

   1. Download vim-master.zip from github/vim/vim
   2. Unzip vim-master.zip.
   3. Then,
```
cd vim-master/src
make -f Make_ming.mak
```
Before the `make` command, I had to edit the file Make_cyg_ming.mak and switch the building rules for pathdef.c. There is a test of whether the shell is sh.exe and different building commands according to this. I have to switch the lines between `ifneq` and `else` with the lines between `else` and `endif`.

Of course, instead of downloading vim-master.zip you might prefer to git-clone the vim repository.

You might receive better instructions from someone else.

I deleted the old html pages because I'm no longer a member of the organization where the pages were and the pages required frequent updates.
